### ISSUE TITLE
Anterograde memory impairment

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6770]
**Preferred term label:**
Anterograde memory impairment

**Synonyms**
Anterograde amnesia

**Definition (free text, please give PubMed ID)**
The impaired ability to establish new long-term memories.
 PMID:24041667 

**Parent term (use hpo.jax.org/app)**
Impaired long-term memory

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
